jetson/percep/pcl.cpp

```diff
@@ -6,16 +6,16 @@
 //Filters out all points with z values that aren't within a threshold
 //Z values are depth values in mm
 //Source: https://rb.gy/kkyi80
```

Comment:
This comment looks out of date... isn't this change in another PR? I feel like I made this comment already.